scripts/qa_crowbarsetup.sh

```diff
@@ -4586,6 +4586,14 @@ function onadmin_prepare_crowbar_upgrade()
         # move nodes to upgrade mode
         safely curl -s -X POST $crowbar_api_digest $crowbar_api/installer/upgrade/prepare.json
     else
+        if crowbarctl upgrade prechecks --format plain | grep "false" ; then
+            complain 11 "Some necessary check before the upgrade has failed"
+        fi
+
+        # FIXME crowbatctl is currently missing admin repo check
+        if curl -s -X GET -H "Accept: application/vnd.crowbar.v2.0+json" $crowbar_api/api/crowbar/repocheck | grep "missing" ; then
```

Comment:
Is it possible to use the new `crowbar_api_request` function here?
If the http code is non [23]xx then the function fails and could thus be used in the if clause.
So the api endpoint needs to return a proper http code - does it do that, @MaximilianMeister ?